tests/agents_t/test_siteagent.py

```diff
@@ -23,6 +23,11 @@ def test_drone_uuid(self):
         self.site_agent.drone_uuid(uuid="test")
         self.site_adapter.drone_uuid.assert_called_with(uuid="test")
 
+    def test_drone_heartbeat_interval(self):
+        self.site_adapter.drone_heartbeat_interval.return_value = 60
```

Comment:
If the default is `60`, don't you want to set it to another value in the test?

Reply:
This code is actually mocking the default of 60s, since no real `SiteAdapter` is called in the test.

Reply:
So you want to test, if the default works. Wouldn't be the test more general, if you would chose any value?

Reply:
No, I want to mock the default behaviour of the `SiteAdapter` and ensure that the right property is called.  The actual test of the configurable parameter is in the interface. https://github.com/MatterMiners/tardis/blob/e4e35068c1c083a40ef0f0a878bd9f5c93e6545f/tests/interfaces_t/test_siteadapter.py#L101-L105

Reply:
Okay so in this context `60` is any.